src/test/java/teammates/test/driver/TimeHelperExtension.java

```diff
@@ -66,4 +68,22 @@ public static int convertToOptionValueInTimeDropDown(LocalDateTime localDateTime
         }
         return hour;
     }
+
+    /**
+     * Returns the date object with specified offset in number of ms from now.
+     * @deprecated Use {@link TimeHelperExtension#getInstantMillisOffsetFromNow(long)} instead.
+     */
+    @Deprecated
+    public static Date getMsOffsetToCurrentTime(long offsetMilliseconds) {
```

Comment:
Since we're already touching the users of this method in moving it to `TimeHelperExtension`, can we omit this method and update the users to use the new method?